core/system/src/mpu/vmpu.c

```diff
@@ -423,21 +423,20 @@ int vmpu_box_id_self(void)
 
 int vmpu_box_id_caller(void)
```

Comment:
Just a note: I expect this function to be deleted when we remove secure gateways.